testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oidc/AudienceTest.java

```diff
@@ -74,7 +74,7 @@ public void configureTestRealm(RealmRepresentation testRealm) {
 
         // Create sample user
         UserRepresentation user = UserBuilder.create()
-                .id(userId)
+                .id(KeycloakModelUtils.generateId())
```

Comment:
@hmlnarik Does this line differ then in the class: ResourceOwnerPasswordCredentialsGrantTest.java that id is not deleted but rather generated?

Reply:
No, they are the same in essence